ocaml/idl/schematest.ml

```diff
@@ -1,7 +1,7 @@
 let hash x = Digest.string x |> Digest.to_hex
 
 (* BEWARE: if this changes, check that schema has been bumped accordingly *)
-let last_known_schema_hash = "5115945f27703c84a08dcc94a10469e5"
+let last_known_schema_hash = "862ad9d640be8f53ec8c1b3085b83332"
```

Comment:
Since the schema has changed, the schema version should be bumped (`grep -re 'let schema_.*_vsn'
`)

Reply:
There was a schema change in feature branch.
This is another change on top of that one. I would expect that in merging into master, there would be only one bumping-up on the schema version. So I don't bump it here.

Reply:
Okay, my bad. In that case we'll leave it